org.osate.ui/src/org/osate/ui/wizards/NewAadlPackageWizard.xtend

```diff
@@ -68,6 +81,50 @@ class NewAadlPackageWizard extends AbstractNewFileWizard {
 		]
 	}
 	
+	def private findPackageInScope(IContainer parent, String packageName) {
```

Comment:
New method for looking up a package name in the scope of the project that *is* or contains `parent`.